packages/e2e/test/web-extension/extension/ui.ts

```diff
@@ -244,3 +251,27 @@ document.querySelector('#buildAndSignTx')!.addEventListener('click', async () =>
   const { tx: signedTx } = await builtTx.sign();
   setSignature(signedTx.witness.signatures.values().next().value);
 });
+
+// Code below tests that a disconnected port in background script will result in the consumed API method call promise to reject
+// UI consumes API -> BG exposes fake API that closes port
+const disconnectPortTestObj = consumeRemoteApi(
```

Comment:
Probably time to refactor ui.ts/ui.html to group by what it's testing. It's ok to create a separate task for it.

Reply:
added a task for it